src/hooks/useSelector.js

```diff
@@ -10,7 +10,7 @@ function useSelectorWithStoreAndSubscription(
   selector,
   equalityFn,
   store,
-  contextSub
+  contextSub,
```

Comment:
Probably don't want this.
```suggestion
  contextSub
```